src/codegen/mod.rs

```diff
@@ -2294,6 +2412,38 @@ mod utils {
     use syntax::ast;
     use syntax::ptr::P;
 
+
+    pub fn prepend_objc_header(ctx: &BindgenContext,
+                               result: &mut Vec<P<ast::Item>>) {
+
+
+        let use_objc;
+
+        if ctx.options().objc_extern_crate {
```

Comment:
At first I wrote it like that, and I now put it back like that. Somehow the style doesn't quite appeal to me, especially if the condition is long and/or the blocks are sprawling. I think I got it decent now with rustfmt disabled, which made the unwrap hanging awkwardly.